tests/Core/Tokenizers/Tokenizer/RecurseScopeMapSwitchTokenScopeTest.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/* testSwitchNormalSyntax */
+switch ($value) {
+    case 1:
+        echo 'one';
+        break;
+    default :
+        echo 'other';
+        break;
+}
+
+// Test for https://github.com/squizlabs/PHP_CodeSniffer/issues/497
+/* testSwitchAlternativeSyntax */
+switch ($value):
+    /* testEnsureTestWillNotPickUpWrongColon */
```

Comment:
Suggestion: `testSwitchAlternativeSyntaxEnsureTestWillNotPickUpWrongColon`.

I realize that makes it a very long name, but it will reduce the chance of there being duplicate markers once more tests get added to the file and it closely ties the marker name to the code sample test marker name, making it clear these test markers belong together and are related to the same code sample.

Reply:
Done